charon/src/values_utils.rs

```diff
@@ -218,6 +218,9 @@ impl ScalarValue {
             ScalarValue::I128(v) => v as u128,
         }
     }
+    pub fn from_bits(ty: IntegerTy, bits: u128) -> Self {
```

Comment:
Nit: could you add a space above?

Reply:
FTR I like grouping together functions that belong together

Reply:
I like having spaces, I think it's cleaner :)